eventually/task/models.py

```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
     status = models.IntegerField(default=0, choices=STATUS_TYPE_CHOICES)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
+    users = models.ManyToManyField(CustomUser)
+
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        task instance object.
+
+        :return: task title, task description, task status

```

Comment:
update docstring's `return` statement according to function's return.

Reply:
ok